isb_lib/geome_adapter.py

```diff
@@ -54,6 +55,79 @@ def geomeEventRecordTimestamp(record):
 
 
 class GEOMEIdentifierIterator(isb_lib.core.IdentifierIterator):
+    comparison_date = datetime.datetime(year=2021, month=5, day=15)
+    known_existing_projects = [
```

Comment:
Wanted to get this checked in so we have a record of it for historical purposes, however, this is not part of the plan moving forward.  This list was generated by this SQL query:

```
select distinct(resolved_content->'record'->'projectId') from Thing where authority_id='GEOME';
```